libs/types/src/system_settings.ts

```diff
@@ -148,11 +149,7 @@ export const DEFAULT_MARK_THRESHOLDS: Readonly<MarkThresholds> = {
   writeInTextArea: 0.05,
 };
 
-/**
- * The default bitonal threshold for scanning ballots.
- * See Section 2.1.43 of the PDI PageScan software specification.
- */
-export const DEFAULT_BITONAL_THRESHOLD = 75;
+export const DEFAULT_MINIMUM_DETECTED_BALLOT_SCALE = 0.975;
```

Comment:
I don't remember how we landed on this value but assuming you got it from somewhere

Reply:
Sounds good!